### PR TITLE
remove meta from title

### PR DESCRIPTION
**Pull request by srsuddath**
## Description
This PR adds a function to remove an unnecessary meta info from appearing in search result titles.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#27061

### CODE REVIEW

**Review by kelsonic — APPROVED** [review]

File: src/applications/search/utils.js [hunk 1]
```diff
@@ -12,3 +12,7 @@ export function truncateResponseString(string, maxLength) {
   }
   return `${string.slice(0, maxLength)}...`;
 }
+
+export function removeDoubleBars(string) {
+  return string.replace('| Veterans Affairs', '');
```
Comment (zacharymorel) [inline src/applications/search/utils.js:17]: Where does "| Veterans Affairs" come from?

Comment (srsuddath) [reply src/applications/search/utils.js:17]: I believe it is coming from the metadata description from the CMS

Comment (srsuddath) [reply src/applications/search/utils.js:17]: they are looking at fixing this on their end, but that may or may not happen, and if so they can't promise a timeline

Comment (zacharymorel) [reply src/applications/search/utils.js:17]: Oh I have write many a line of code for this exact reason haha.

**Review by zacharymorel — APPROVED** [review]

**Pull request merged**